rules/autoconsent/ok.json

```diff
@@ -0,0 +1,50 @@
+{
+    "name": "ok",
+    "vendorUrl": "https://ok.ru/",
+    "cosmetic": false,
+    "runContext": {
+        "urlPattern": "^https://ok\\.ru/"
+    },
+    "prehideSelectors": [".cookies-agreement-notification,.modal-new:has([data-module=SetupCookies])"],
+    "detectCmp": [
+        {
+            "exists": ".cookies-agreement-notification"
+        }
+    ],
+    "detectPopup": [
+        {
+            "visible": ".cookies-agreement-notification"
+        }
+    ],
+    "optIn": [
+        {
+            "waitForThenClick": ".cookies-agreement-notification .cb_accept"
+        }
+    ],
+    "optOut": [
+        {
+            "wait": 1000
```

Comment:
Why do we have to wait 1s, when the next step is a `waitForThenClick`?

Reply:
it's the same race condition again: I didn't manage to get a readiness clue from the DOM structure